libs/shared-web/src/hooks/useFeatureFlags.ts

```diff
@@ -24,9 +24,17 @@ export const useFeatureFlags = (organization?: IOrganizationEntity) => {
 };
 
 export const useFeatureFlag = (key: FeatureFlagsKeysEnum): boolean => {
+  const ldFlagSet = useFlags();
   /** We knowingly break the rule of hooks here to avoid making any LaunchDarkly calls when it is disabled */
-  // eslint-disable-next-line
-  const flagValue = checkShouldUseLaunchDarkly() ? useFlags()[key] : undefined;
+  const flagValue = checkShouldUseLaunchDarkly() ? ldFlagSet[key] : undefined;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** this part of the code I see was important, we avoid making LD calls when its disabled, please keep it the old way